src/components/jobDetailsComponent/jobDetails.tsx

```diff
@@ -72,16 +67,13 @@ const JobDetails: React.FC = () => {
               {job.reservedUser === user.id ? (
                 <Button
                   variant="outlined"
-                  onClick={toggleVerificationUpload}
+                  onClick={() => history.push(`/verify/${job.id}`)}
```

Comment:
is there going to be a separate verify component? I thought the verification was gonna be done through the user's account or the job detail page?